SIAB/data/demo.py

```
@@ -0,0 +1,263 @@
+            for a in self.atomspecies:
+                if a.forb is None:
+                    a.forb = a.jygen(orbital_dir)
+                out += f'{a.forb}\n'
+            out += '\n'
+        out += 'LATTICE_CONSTANT\n1.8897259886\n' # 1 Angstrom in a.u.
+        out += 'LATTICE_VECTORS\n'
+        for i in range(3):
+            out += ' '.join([str(x) for x in self.cell[i]]) + '\n'
+        out += 'ATOMIC_POSITIONS\nCartesian_angstrom_center_xyz\n'
+        
+        idx = np.argsort(self.type_map)
```

Ruff (0.8.2)

Local variable `idx` is assigned to but never used

Remove assignment to unused variable `idx`

(F841)